packages/sdks/e2e/qwik-city/package.json

```diff
@@ -14,11 +14,11 @@
     "start": "vite --open --mode ssr",
     "qwik": "qwik",
     "test": "SERVER_NAME=qwik-city yarn g:nx test:e2e @sdk/tests",
-    "serve": "yarn run dev"
```

Comment:
oh god, that's been an oversight for a long time 😬

Reply:
yess! didn't know that there was a HUGE difference with `--mode ssr` (yarn dev) vs yarn preview in case of qwik